docs/installation.rst

```diff
@@ -9,12 +9,11 @@ Stable release
 --------------
 
 Because RavenPy relies on the `Raven <http://raven.uwaterloo.ca>`_ binary (which is downloaded and
-compiled during the setup), it must be installed in a virtual
+compiled during the setup), it is preferable to install it in a virtual
 environment (either a classic `Python venv
 <https://docs.python.org/3/tutorial/venv.html>`_ or a `Conda
 environment
-<https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_
-(which must be different than the ``base`` environment).
```

Comment:
Just curious, why remove this notice "which must be different than the ``base`` environment" ?

Reply:
In general, it's not great practice to muck up the `base` environment. It's the same principle as the demand for a `venv` but applied to Anaconda.

Reply:
Yes good catch. The reason for this particular change is that this PR removes the notion of "focing the user of being in a venv (conda or plain)". Since there is no such verification, it means that I no longuer have to check if the conda env is the base one.

This is a complicated thought process (of mine). My primary reason for this PR was the RTD build: it failed because if wasn't in a venv. So I revisited the install process and came up with this PR. So I discovered something new:

https://github.com/CSHS-CWRA/RavenPy/blob/setup-without-venv/setup.py#L94-L97

So even though the fact we are removing the venv constraint might result in unwanted results in some specific contexts (such as the user installing with sudo and his system python) I think this PR still provides reasonable outcomes (i.e. the binaries installed in the venv/bin or ~/.local/bin) in most cases.

What do you think? This is all very experimental to me, I might be wrong for many assumptions.

Reply:
Still not 100% clear what your RTD build problem is but I can say that for Raven, the Makefile targets do not have dependencies on Conda env creation targets anymore.

Basically it assumes user already managed to have a working env.  I don't remember if there is detection for user not in any kind of venv which would allow modifying the system python, which is bad.

However, make targets to create the conda env is left there for user to call manually if desired (saving the user from remembering the exact conda syntax and avoid using the `base` conda env).

So basically, no direct management of pip/conda venv is OK with me.  We should leave enough help behind to assist user to create the venv though, which I think you did via documentation.  I would prefer direct in some code to minimize user error but docs is fine too.

Reply:
The central problem we have to manage in this context is the handling of external binaries (Raven and OSTRICH), that need to be available (on the PATH or otherwise) to RavenPy.